README.md

```diff
@@ -237,8 +237,11 @@ tarantool> function get_cluster_nodes() return { 'host1:3301', 'host2:3302', 'ho
 
 You need to pay attention to a function contract we are currently supporting:
 * The client never passes any arguments to a discovery function.
-* A discovery function _should_ return a single result of strings (i.e. single 
-  string `return 'host:3301'` or array of strings `return {'host1:3301', 'host2:3301'}`).
+* A discovery function _must_ return an array of strings (i.e `return {'host1:3301', 'host2:3301'}`).
+* Each string _should_ satisfy the following pattern `host[:port]`
+  (or more strictly `/^[^\:]+(:\d)?$/` - a mandatory host containing any string
```

Comment:
Nit: A colon is not a special character in PCRE and can be used in brackets w/o escaping.

https://www.regular-expressions.info/characters.html#special